config/splits/vpge/field.field.node.stanford_event.su_vpge_event__image.yml

```diff
@@ -1,20 +1,25 @@
-uuid: 5eaeff98-2e6e-44ea-bfd9-d3e8edf6b336
+uuid: f0178bbe-df2e-49e0-8d01-616dd2150e0e
 langcode: en
 status: true
 dependencies:
   config:
-    - field.storage.node.su_vpge_featured_image
+    - field.storage.node.su_vpge_event__image
     - media.type.image
     - node.type.stanford_event
-id: node.stanford_event.su_vpge_featured_image
-field_name: su_vpge_featured_image
+  content:
+    - 'media:image:77b5efa3-fb2a-43a0-b637-69dd835fa0ea'
+id: node.stanford_event.su_vpge_event__image
+field_name: su_vpge_event__image
```

Comment:
Why changing the name & uuid? this will have negative impacts.

Reply:
We should have leeway in naming conventions since I'm introducing the media field.

Reply:
With Jacob doing the display settings, this would conflict with his work. let's keep it the way it is.